### PR TITLE
Mei wssecurity migfat1015 - wssecurity cxf

### PR DESCRIPTION
**Pull request by meiaus**
Migrate CL FAT:
- com.ibm.ws.wssecurity_fat
- com.ibm.ws.wssecurity_fat.cxf

to OL FAT:
- com.ibm.ws.wssecurity.fat.utils.common
- com.ibm.ws.wssecurity_fat.wsscxf

### CODE REVIEW

**Comment by meiaus** [comment]
#build

**Comment by tevans78** [comment]
> **The following files contain a string in its header indicating IBM copyrighted content:
> 
> **- dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/derivekey/src/com/ibm/ws/wssecurity/fat/derivekey/DeriveKeyWebSvc1.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/mustunderstand/src/com/ibm/ws/wssecurity/fat/mustunderstand/MustUnderstandWebSvc6.java
> 
>     * dev/com.ibm.ws.wssecurity.fat.utils.common/fat/src/com/ibm/ws/wssecurity/fat/utils/common/SharedTools.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/mustunderstand/src/com/ibm/ws/wssecurity/fat/mustunderstand/MustUnderstandWebSvc4.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/mustunderstand/src/com/ibm/ws/wssecurity/fat/mustunderstand/MustUnderstandWebSvc7.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/.settings/org.eclipse.jdt.ui.prefs
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/derivekey/src/com/ibm/ws/wssecurity/fat/derivekey/DeriveKeyWebSvc2.java
> 
>     * dev/com.ibm.ws.wssecurity.fat.utils.common/.settings/org.eclipse.jdt.ui.prefs
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/mustunderstand/src/com/ibm/ws/wssecurity/fat/mustunderstand/MustUnderstandWebSvc3.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/mustunderstand/src/com/ibm/ws/wssecurity/fat/mustunderstand/MustUnderstandWebSvc2.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/derivekey/src/com/ibm/ws/wssecurity/fat/derivekey/DeriveKeyWebSvc5.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/derivekey/src/com/ibm/ws/wssecurity/fat/derivekey/DeriveKeyWebSvc4.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/mustunderstand/src/com/ibm/ws/wssecurity/fat/mustunderstand/MustUnderstandWebSvc1.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/derivekey/src/com/ibm/ws/wssecurity/fat/derivekey/DeriveKeyWebSvc6.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/derivekey/src/com/ibm/ws/wssecurity/fat/derivekey/DeriveKeyWebSvc3.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/mustunderstand/src/com/ibm/ws/wssecurity/fat/mustunderstand/MustUnderstandWebSvc8.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/derivekey/src/com/ibm/ws/wssecurity/fat/derivekey/DeriveKeyWebSvc7.java
> 
>     * dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/mustunderstand/src/com/ibm/ws/wssecurity/fat/mustunderstand/MustUnderstandWebSvc5.java

@meiaus Please remove IBM Confidential copyright headers ASAP

**Comment by tevans78** [comment]
#run-libby-bot

**Comment by tevans78** [comment]
> **The following files contain a string in its header indicating IBM copyrighted content:
> 
> **- dev/com.ibm.ws.wssecurity_fat.wsscxf/.settings/org.eclipse.jdt.ui.prefs
> 
>     * dev/com.ibm.ws.wssecurity.fat.utils.common/.settings/org.eclipse.jdt.ui.prefs

@meiaus Please remove IBM Confidential copyright header from the code template as well

**Comment by arunavemulapalli** [comment]
#build

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.wsscxf

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Comment by meiaus** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.wsscxf

**Comment by meiaus** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.wsscxf

**Comment by arunavemulapalli** [comment]
#run-libby-bot

File: dev/com.ibm.ws.wssecurity_fat.wsscxf/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
```
Comment (arunavemulapalli) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf/bnd.bnd:3]: update Copyright year

File: dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/cxfclient/resources/WEB-INF/web - Copy.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <servlet-name>cxfclient</servlet-name>
+        <url-pattern>/CxfNoWssecSvcClient</url-pattern>
+    </servlet-mapping>
+
+    <session-config>
+        <session-timeout>60</session-timeout>
+    </session-config>
+
+</web-app>
+
```
Comment (arunavemulapalli) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/cxfclient/resources/WEB-INF/web - Copy.xml:32]: we can remove this file

File: dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/nopassclient/resources/WEB-INF/web - org.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <servlet-name>nopassclient</servlet-name>
+        <url-pattern>/CxfNoPassSvcClient</url-pattern>
+    </servlet-mapping>
+    
+    <session-config>
+        <session-timeout>60</session-timeout>
+    </session-config>
+
+</web-app>
+
```
Comment (arunavemulapalli) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf/test-applications/nopassclient/resources/WEB-INF/web - org.xml:32]: we can remove this

**Comment by neuwerk** [comment]
#run-libby-bot

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Comment by arunavemulapalli** [comment]
#build

**Review by arunavemulapalli — APPROVED** [review]

**Comment by arunavemulapalli** [comment]
latest pb is good except couple of unrelated ui test failures

**Pull request merged**